org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/dom/DOMNode.java

```diff
@@ -68,6 +68,8 @@
 
 	DOMNode parent;
 
+	private static XMLNodeList<DOMNode> EMPTY_CHILDREN = new XMLNodeList<DOMNode>();

```

Comment:
static final?

Reply:
be careful this object should be immutable

Reply:
@fbricon updated